ctrlf.el

```diff
@@ -458,6 +459,17 @@ direction is backwards."
   (set-window-point (minibuffer-selected-window) ctrlf--starting-point)
   (abort-recursive-edit))
 
+(defun ctrlf-insert-symbol-at-point ()
+  "Use symbol at point to replace user input."
+  (interactive)
+  (let ((symbol (with-selected-window
+                    (minibuffer-selected-window)
+                  (thing-at-point 'symbol t))))
+    (delete-minibuffer-contents)
+    (if ctrlf--regexp-p
+        (insert (concat "\\_<" symbol "\\_>"))
+      (insert symbol))))
```

Comment:
This should have a `regexp-quote`, right?

Reply:
Done.